src/detours.cpp

```diff
@@ -18,6 +18,22 @@
 
 #define NOTHROW
 
+//////////////////////////////////////////////////////////////////////////////
+//
+
+extern "C" void __cdecl _assert(const char* expr, const char* filename, unsigned lineno);
+void DetourAssert(const char* expr, const char* filename, unsigned lineno)
```

Comment:
You introduced a seperate Assert method in one of your other PRs, can you figure out which one you want to contribute and stanardize on that?

Reply:
This one, the one in the other PR can only be used in the _DEBUG environment, and this one is unlimited. Can also be used in a publishing environment. It's just that the commit was earlier, it should be covered by this, guess there should be no conflict, if there is a merge conflict, you can tell me, and I will change it.